blocks/eloqua-form/forms/servicesubscriptions.html

```diff
@@ -110,7 +110,7 @@
   }
 
   .elq-form body {
-    font-family: Helvetica Neue, Helvetica, Arial, sans-serif;
+    font-family: Helvetica Neue, Helvetica, "Noto Sans", sans-serif;
```

Comment:
If I recall correctly, Eloqua Forms was discontinued, so, in fact, it doesn't matter because is legacy code. I just changed all *Arial* Reference. Besides, these HTML are form templates that are not too important to format properly.

Reply:
also check [StackOverflow](https://stackoverflow.com/questions/53846582/can-i-use-css-variables-in-a-font-list-and-have-it-work-in-legacy-browsers) it says that we can't use variables for font-face or font-family